src/Arcus.Templates.WebApi/.template.config/template.json

```diff
@@ -87,6 +95,10 @@
     "CertificateAuth": {
       "type": "computed",
       "value": "(authentication == \"Certificate\")"
-    } 
+    },
+    "AppSettings": {
+      "type": "computed",
+      "value": "CertificateAuth"
```

Comment:
Not a template expert, but is this correct?

Reply:
This means we have a 'computed' directive that will result in TRUE when the certificate authentication is TRUE.
'Computed' means not configurable for the user but generated at creation-time.

Reply:
So, yes; correct. :)